### PR TITLE
Add missing app fields

### PR DESCRIPTION
**Pull request by thibaultleouay**
Add missing fields from app https://wit.ai/docs/http/20170307#get__apps__app_id_link

### CODE REVIEW

File: apps_test.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func TestGetApps(t *testing.T) {
 
 func TestGetApp(t *testing.T) {
 	testServer := httptest.NewServer(http.HandlerFunc(func(res http.ResponseWriter, req *http.Request) {
-		res.Write([]byte(`{"name": "alarm-clock"}`))
+		res.Write([]byte(`{"name": "alarm-clock","training_status":"done"}`))
```
Comment (plutov) [inline apps_test.go]: Please add here all fields which are possible to check that unmarshalling works well. Especially, please check if `time.Time` is converted correctly.

**Comment by thibaultleouay** [comment]
I had to create a custom type for the time as the time sent by the wit.ai api is not in the rfc3339 format.

**Comment by plutov** [comment]
Nice, thank you for working on this!

**Comment by thibaultleouay** [comment]
The integration test TestIntegrationSamples also fail on master

**Comment by plutov** [comment]
@thibaultleouay master looks green - https://travis-ci.org/wit-ai/wit-go/branches. Have you tried it locally, you need to set WITAI_INTEGRATION_TOKEN locally.

**Comment by thibaultleouay** [comment]
Locally with my WITAI_INTEGRATION_TOKEN it works. 
Also on master I dont know how they passed. 

The sample test makes  this request when you dont set any SampleEntity
`
  curl -XPOST 'https://api.wit.ai/samples?v=20170307' \
  -H "Authorization: Bearer $YOUR_TOKEN" \
  -H "Content-Type: application/json" \
  -d '[{
        "text": "I live in London",
        "entities": null
      }]'`

**Comment by plutov** [comment]
Agree, locally it works well, I am merging this PR, and will investigate the tests after.

**Pull request merged**